product_custom_info/README.rst

```diff
@@ -1,77 +0,0 @@
-.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
-    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
-    :alt: License: LGPL-3
-
-===================
-Product Custom Info
-===================
-
-This module extends products to allow having custom properties on them.
-
-Installation
-============
-
-* You depend from module ``base_custom_info``, found in `OCA/server-tools
```

Comment:
@ilyasProgrammer If you remove the README you need to fill in all the corresponding readme fragments (INSTALL/CONFIGURE/CONTRIBUTORS...).